source/helix-ui.less

```diff
@@ -4,8 +4,8 @@
 @import 'imports';
 
 // LESS Vars and Mixins
-@import (less, reference) 'vars';
-@import (less, reference) 'mixins';
```

Comment:
`less` is unnecessary, because it is the default if the file name doesn't have an extension.